hadoop-connector-examples/README.md

```diff
@@ -118,9 +118,9 @@ Hadoop (verified with Hadoop 2.8.1 and 3.1.1 on Ubuntu 16.04)
    export HADOOP_EXAMPLES_OUTPUT=${HDFS}/tmp/hadoop_examples_output
    export PRAVEGA_URI=tcp://<pravega_controller_ip_and_port> # e.g. tcp://192.168.0.188:9090
    export PRAVEGA_SCOPE=<scope_name>   # e.g. myScope
-   export PRAVEGA_INPUT_STREAM=<stream_name> # e.g. intStream
-   export PRAVEGA_OUTPUT_STREAM=<stream_name> # e.g. outStream
-   export PRAVEGA_OUTPUT_STREAM_PREFIX=<stream_name_prefix> # e.g. sortedStream
+   export PRAVEGA_STREAM=<stream_name> # e.g. myStream
+   export PRAVEGA_OUTPUT_STREAM_PREFIX=<stream_prefix_name> # e.g. outputStreamPrefix
+   export PRAVEGA_NUMBER_OF_SEGMENTS=<number_of_segments> # e.g. 12
```

Comment:
This is per Stream or in total? Maybe we could specify that in the variable name.